extensions/2.0/Khronos/KHR_materials_sss/schema/glTF.KHR_materials_sss.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_sss glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the parameters for the volume of a material.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "scatterDistance": {
+          "type": "number",
+          "minimum": 0.0,
```

Comment:
scatterDistance should declare the minimum as an `exclusiveMinimum`.  See the schema for [KHR_materials_volume](https://github.com/KhronosGroup/glTF/blob/c6a5521ebd1469aba076949a97f150bf3591b63f/extensions/2.0/Khronos/KHR_materials_volume/schema/glTF.KHR_materials_volume.schema.json#L23)